src/server/api/admin_api.py

```diff
@@ -61,15 +68,32 @@ def execute():
     statistics = get_statistics()
 
     last_execution_details = {"executionTime": current_time, "stats": statistics}
+    last_ex_json = (json.dumps(last_execution_details))
 
-    last_execution_file = open(LOGS_PATH + "last_execution.json", "w")
-    last_execution_file.write(json.dumps(last_execution_details))
-    last_execution_file.close()
+    # Write Last Execution stats to DB  
+    # See Alembic Revision ID: 05e0693f8cbb for table definition
+    with engine.connect() as connection:
+        ins_stmt = insert(kvt).values(               # Postgres-specific insert() supporting ON CONFLICT 
+            keycol = 'last_execution_time',
+            valcol = last_ex_json,
+            )
+        # If key already present in DB, do update instead 
+        upsert = ins_stmt.on_conflict_do_update(
+                constraint='kv_unique_keycol_key',
+                set_=dict(valcol=last_ex_json)
+                )
+
+        try:
+            connection.execute(upsert)
+        except Exception as e:
```

Comment:
didn't work for me when I ran execute. didn't see and data being stored in the table.

Reply:
Are you running locally or in Docker? If locally, did you run `alembic upgrade head`? Is the kv_unique table present in the DB?

Reply:
The value in the `alembic_version` table should be 05e0693f8cbb.